### PR TITLE
Portal 1012/readme docs

### PR DESCRIPTION
**Pull request by cfarmer-fearless**
Main README documentation for using and developing this application.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,86 @@
...
+
+`swa start http://localhost:5173 --run "yarn dev" --api-location ./api`
+
+This command runs the Vite dev server for the front end app on `localhost:5173`, and subsequently the Azure function simulator for the backend on `localhost:7071`. Both of these hosts get automatically proxied under a single host at `localhost:4280`.
+
+**Note: While this is the recommended way to set up the local dev environment, you will see a degrade in performance in the app. We believe this is due to poor performance in the the SWA CLI proxy.**
+
+### Using separate commands
+
+Of course, it is possible to run the front end and backend dev servers separatly if you do not with to use the SWA CLI. However, we do not recommend doing this as it requires CORS to be configured in the backend, and environment variables to be changed. If you still with to do this, you need to run the following commands to start each server:
```
Comment (sublettk) [inline README.md:72]: typo: wish
'if you still with to do this` --> 'if you still wish to do this'

File: README.md [hunk 2]
```diff
@@ -1,10 +1,86 @@
...
+- Start the front end server by running `yarn dev` in the project root directory.
+- Start the backend server by running `func host start` in the `api` directory.
+
+## Running Automated Tests
+
+We encourage TDD while making changes to this app. To run unit tests for the front end, simply run `yarn test` in the project root directory. These tests will be run as part of the acceptance criteria for a PR.
+
+## Manual Deployments to the Dev Environment
+
+This app is deployed to a single Azure Static Web instance. This instance has a dev, staging, and production environment. Eventually, a CD pipeline will be built to automate the deployment process to each environment. **Manual deployments should only be used when this pipeline is unavailable.** The following instructions will explain how to perform a manual deployment to the dev environment.
```
Comment (sublettk) [inline README.md:83]: Shall we create a backlog item for this pipeline? If so, let's include updating this section of the doc as part of the DoD.

**Review by sublettk — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (cfarmer-fearless) [reply README.md:83]: Good call!  Issue created: https://cdc-dexops.atlassian.net/browse/PORTAL-1027

**Review by sublettk — APPROVED** [review]
LGTM!

**Pull request merged**